pkg/ecdsa/tss/tss_test.go

```diff
@@ -127,7 +127,7 @@ func TestGenerateKeyAndSign(t *testing.T) {
 
 	firstSigner := signers[groupMemberIDs[0].String()]
 	firstPublicKey := firstSigner.PublicKey()
-	curve := secp256k1.S256()
+	curve := S256()
```

Comment:
Same here re: having a comment indicating where to find this function + why we're using it, I think?

Reply:
Please see https://github.com/keep-network/keep-ecdsa/commit/382ba37582d155d0735476a65518b831f17dca52